client/ionic/src/pages/CheckYourHealth.tsx

```diff
@@ -11,13 +10,12 @@ import {
 } from '@ionic/react';
 import TopNav from '../components/TopNav';
 import 'tachyons';
-import { getUserContext } from '../content/userContext';
 import useDynamicFlow from '../hooks/useDynamicFlow';
 
-const CopingParents: React.FC = () => {
+const CheckYourHealth: React.FC = () => {
   // TODO: Refactor this out to separate Flow components. Use a dictionary
   // of screen archetypes.
-  const flow = useDynamicFlow('copingParents');
+  const flow = useDynamicFlow('check');
```

Comment:
I'm worried about future collisions on the name `check` for the flow, let's choose something more specific to this page

Reply:
we should use a standard atm of react name converted to kebab case is flow.

Reply:
done